dataland-backend/src/main/kotlin/db/migration/V2__GenerateCompanyNameIdentifierIndex.kt

```diff
@@ -0,0 +1,31 @@
+package db.migration
+
+import org.flywaydb.core.api.migration.BaseJavaMigration
+import org.flywaydb.core.api.migration.Context
+
+/**
+ * This migration script creates an index for a faster search for company names and identifiers
+ */
+@Suppress("ClassNaming")
+class V2__GenerateCompanyNameIdentifierIndex : BaseJavaMigration() {
+    override fun migrate(context: Context?) {
+        context!!.connection.createStatement().execute(
+            "CREATE EXTENSION IF NOT EXISTS pg_trgm; " +
+                "CREATE INDEX IF NOT EXISTS idx_company_identifier " +
```

Comment:
As discussed the Flyway way of doing this would be without the _IF NOT EXISTS_

Reply:
Aaah right. On it.

Reply:
Changed it. However I want to test if this works. Also we should set the baseline version on preview to 2 now. Since it has an index already. But I'll check.